### PR TITLE
COM-193 Update list of HIV ARV regimen, Added drug concepts and new drug csv. Added script for remove unwanted drugs.

### PR DESCRIPTION
**Pull request by IipperMangesh**
Added Concepts and Drugs in metadata sharing for all indicators.

### CODE REVIEW

**Comment by tmvumbi2** [comment]
@IipperMangesh could you perhaps use a more specific name for the concept csv file? I think something like `drugs.csv` will do

File: configuration/concepts/concepts_base.csv [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Uuid,Void/Retire,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,_version:1,_order:1000
```
Comment (tmvumbi2) [inline configuration/concepts/concepts_base.csv]: @IipperMangesh there is no value populated for the column `Void/Retire`; if the default value is 'false`, could you please check if it's possible not to include this column entirely?

File: configuration/concepts/concepts_base.csv [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+0aca2d79-1a16-42c8-b2c0-a4989bd55aec,,AZT+3TC+ATZ/r,,AZT+3TC+ATZ/r,,AZT+3TC+ATZ/r,,Drug,N/A,,
```
Comment (tmvumbi2) [inline configuration/concepts/concepts_base.csv]: @IipperMangesh the import failed on the above concept (see below the error message from the log); I suspect this is due to the double commas

`
ERROR - CsvParser.saveAll(184) |2019-04-30 07:44:09,986| An OpenMRS object could not be constructed or saved from the following CSV line: [0aca2d79-1a16-42c8-b2c0-a4989bd55aec, null, AZT+3TC+ATZ/r, null, AZT+3TC+ATZ/r, null, AZT+3TC+ATZ/r, null, Drug, N/A, null, null]
`

File: configuration/concepts/concepts_base.csv [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+48728efa-b311-4696-a8d0-f2bc155ccd35,,ABC+3TC+ATZ/r ,,ABC+3TC+ATZ/r ,,ABC+3TC+ATZ/r ,,Drug,N/A,,
```
Comment (tmvumbi2) [inline configuration/concepts/concepts_base.csv]: @IipperMangesh Please ensure that there is no trailing space after the concept's name. Could you please also trim the concept names on the lines below as well?

File: configuration/concepts/concepts_base.csv [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+b88c790f-f0db-4510-8e3c-9a7e012ef4b3,,TDF+3TC+ATZ/r ,,TDF+3TC+ATZ/r ,,TDF+3TC+ATZ/r ,,Drug,N/A,,
+1a6cdf69-1472-4b4c-9f57-e12db2681ee7,,TDF+3TC+LPVr ,,TDF+3TC+LPVr ,,TDF+3TC+LPVr ,,Drug,N/A,,
+a8f47b0b-1dde-4adf-8707-879b26159de5,,AZT+DRV+RTV ,,AZT+DRV+RTV ,,AZT+DRV+RTV ,,Drug,N/A,,
+a6aeea7f-620b-424e-a68e-38b28f4ed588,,ABC+DRV+RTV,,ABC+DRV+RTV,,ABC+DRV+RTV,,Drug,N/A,,
+66e89a39-4aa8-4d8b-b7b2-704fc678bc25,,AZT/3TC+EFV,,AZT/3TC+EFV,,AZT/3TC+EFV,,Drug,N/A,,
```
Comment (tmvumbi2) [inline configuration/concepts/concepts_base.csv]: @IipperMangesh This drug exists already in the current dev box and it has a concept associated with it.
The drugs from line 9 to 15 also exist already

File: configuration/concepts/concepts_base.csv (hunk 2 above)
Comment (tmvumbi2) [reply configuration/concepts/concepts_base.csv]: @IipperMangesh the concept names should be repeated 4 times on each row (full specified name in fr and en, and short name in fr and en); they are currently repeated 3 times

**Review by tmvumbi2 — CHANGES_REQUESTED** [review]

**Comment by tmvumbi2** [comment]
@IipperMangesh please note the following requirement from the ticket

image

**Comment by prashantsable1991** [comment]
As discussed with @IipperMangesh I only checked for newly created drugs. And new drugs are adding properly.

**Pull request merged**